standardguess

```diff
@@ -0,0 +1,382 @@
+#!/bin/bash
+#checks to see if a video's properties are consistent with a set of standards and if so it assigns expected values to other properties if they are undefined or differ from expectations 
+
+SCRIPTDIR=$(dirname $(which "${0}"))
+. "${SCRIPTDIR}/mmfunctions" || { echo "Missing '${SCRIPTDIR}/mmfunctions'. Exiting." ; exit 1 ;};
+
+_get_frame_rate(){
+    FRAMERATE=$(ffprobe -i "${1}" -v error -select_streams v:0 -show_entries stream=avg_frame_rate -of default=noprint_wrappers=1:nokey=0 | grep "^avg_frame_rate=" | cut -d = -f 2)
```

Comment:
should add a dependency check on the tools used, such as ffprobe, mkvinfo, and mkvpropedit. The dependencies are listed in an array such as https://github.com/mediamicroservices/mm/blob/437c6c71154dd40f7ab15c2da269f13422b98117/fix_rewrap#L6 and then checked with a function from mmfunction such as https://github.com/mediamicroservices/mm/blob/437c6c71154dd40f7ab15c2da269f13422b98117/fix_rewrap#L21

Reply:
Also is `-v error` useful here, it may just make the output noisy.

Reply:
If you set `nokey=1` then you won't need to pipe to grep and cut to parse the value.

Reply:
I think that -v error tells ffprobe to report errors only. Without it prints out all the version info for ffmpeg, ffprobe, etc...